### PR TITLE
add ROCm 3.9 manywheel build

### PR DESCRIPTION
**Pull request by jeffdaily**

### CODE REVIEW

**Comment by jeffdaily** [comment]
@walterddr could you please review?  Thanks.

File: manywheel/build_rocm.sh [hunk 1]
```diff
@@ -59,22 +59,31 @@ elif [[ "$OS_NAME" == *"Ubuntu"* ]]; then
     LIBELF_PATH="/usr/lib/x86_64-linux-gnu/libelf.so.1"
 fi
 
+# rocm3.8 and later use TensileLibrary.dat
 if [[ $ROCM_VERSION == "rocm3.7" ]]; then
     TENSILE_LIBRARY_NAME=TensileLibrary.yaml
-elif [[ $ROCM_VERSION == "rocm3.8" ]]; then
+else
```
Comment (walterddr) [inline manywheel/build_rocm.sh:65]:
```suggestion
elif [[ $ROCM_VERSION == "rocm3.8" || $ROCM_VERSION == "rocm3.9" ]]; then
```

Comment (jeffdaily) [reply manywheel/build_rocm.sh:65]: I did consider that, but IIUC the tensile library name extension will hereafter always be *.dat, so the rocm3.7 case was the only exceptional case.

Comment (walterddr) [reply manywheel/build_rocm.sh:65]: got it. thanks for the explanation

**Review by walterddr — APPROVED** [review]

**Comment by walterddr** [comment]
@jeffdaily did you have rocm3.9 docker images created?

**Pull request merged**